entrypoint.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
I think it is better to move container related stuff into a dedicated `containers/` or alike

Reply:
Sure, can do that, but the Dockerfile should stay at the root for the full context of the repo.